x-pack/plugins/security_solution/cypress/e2e/cases/creation.cy.ts

```diff
@@ -69,7 +69,7 @@ describe('Cases', () => {
   });
 
   it('Creates a new case with timeline and opens the timeline', function () {
-    login();
+    loginWithUser({ username: 'elastic', password: 'changeme' });
```

Comment:
This test was failing because it was logging in as user system_indices users. The tests expects that it is logged in as the `elastic` user.